api/documents/schemas.py

```diff
@@ -0,0 +1,26 @@
+from ninja import Schema
+from typing import List
```

Comment:
**nitpick:** since 3.10 (or 3.11, i don't remember) it's not necessary, you can just type stuff as `list[float]` instead (lowercase `l` without import)